release/preferences.js

```diff
@@ -21,7 +21,7 @@
 // Current prefs
 var prefs = {};
 let debugPrefs =
-    {"debug":true,"firsttime":false,"forcesubmenu":false,"hideopen":false,"hidetab":false,"inbackground":false,"lastversion":"3.0.0","newwindow":false,"topmenu":true,"submenus":["--DIAGNOSTIC MODE--","&www.*","www.*.&com","www.*.&org","www.*.&net","&ftp.*","--","In &Google|http://www.google.com/search?q=*&source-id=mozilla%20firefox&start=0","In Wi&kipedia|http://en.wikipedia.org/wiki/special:search?search=*&sourceid=mozilla-search"],"sandr":["^//||file:///","^([A-Za-z]:)||file:///$1"]};
```

Comment:
FYI this copy of the prefs. is only for local testing of the prefs page logic;  it's only there to populate the HTML if the page is loaded locally into FF.  But yes,m I'd change it for consistency.

Reply:
Thanks for the information and full respect for your decision. :)